src/main/scala/encry/view/wallet/AccountManager.scala

```diff
@@ -62,6 +66,15 @@ case class AccountManager private(store: Store, password: String, mandatoryAccou
 }
 
 object AccountManager {
+
+  def init(mnemonicKey: String, pass: String, settings: EncryAppSettings): Unit = {
+    val keysDir: File = getKeysDir(settings)
+    keysDir.mkdirs()
+    val accountManagerStore: LSMStore = new LSMStore(keysDir, keepVersions = 0, keySize = 34)
+    val account = AccountManager.apply(accountManagerStore, pass, Some(mnemonicKey), 0.toByte)
```

Comment:
you can change Option[seed] to seed and remove logic with new random seed generation

Reply:
!